lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -228,7 +232,7 @@ function dosomething_user_form_alter(&$form, $form_state, $form_id) {
       // Helper text & additional data.
       _dosomething_user_add_signup_data($form);
       $form['#submit'][] = 'dosomething_user_login_submit';
-
+      $_SESSION['log_login'] = "login";
```

Comment:
Can we call this something like dosomething_log_login, just so that we know it's something we've put in via a dosomething module?